src/server/src/main/java/io/cassandrareaper/core/RepairType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024-2024 DataStax, Inc.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cassandrareaper.core;
+
+public enum RepairType {
+    SUBRANGE_FULL("subrange_full"),
+    INCREMENTAL("incremental"),
+    SUBRANGE_INCREMENTAL("subrange_incremental");
+
+  private final String repairType;
+
+  RepairType(String repairType) {
+    this.repairType = repairType;
+  }
+
+  public String getRepairType() {
+    return repairType;
```

Comment:
```suggestion
    return name().toLowercase();
```
(and remove the field) …? 


and fromName just
```
public static RepairType fromName(String name) {
    return valueOf(name.toUppercase());
}
```

Reply:
sounds good 👍